### PR TITLE
Documentation: fix grammatical mistakes

### PR DESCRIPTION
**Pull request by Tejas2805**
```
There are a few grammatical mistakes on the different 
web pages of the RepoSense website.

Let's fix the remaining grammatical errors.
```

### CODE REVIEW

**Review by fzdy1914 — APPROVED** [review]

**Comment by Tejas2805** [comment]
@jamessspanggg Need your review.

**Comment by fzdy1914** [comment]
@jamessspanggg Your idea is needed here.

File: docs/dg/report.md [hunk 1]
```diff
@@ -48,37 +48,37 @@ The tabbed interface is responsible for loading various modules such as authorsh
...
 
 <!-- ==================================================================================================== -->
 
 ## Main ([main.js](https://github.com/reposense/RepoSense/blob/master/frontend/src/static/js/main.js))
 
-This contains the logic for main VueJS object, `window.app`, which is responsible for passing the necessary data into the relevant modules to be loaded.
+This contains the logic for the main VueJS object, `window.app`, which is responsible for passing the necessary data into the relevant modules to be loaded.
 
-`v_summary`, `v_authorship`, `v_zoom`, `v_segment` and `v_ramp` are components which will be embedded into report and will render the corresponding content based on the data passed into it from the main `window.app`.
+`v_summary`, `v_authorship`, `v_zoom`, `v_segment`, and `v_ramp` are components embedded into the report and will render the corresponding content based on the data passed into it from the main `window.app`.
```
[the remark this answers is not in the document]
Comment (Tejas2805) [reply docs/dg/report.md:60]: Yup, it is optional. Want me to remove?

File: docs/dg/architecture.md [hunk 2]
```diff
@@ -64,18 +64,18 @@
 
 ## ReportGenerator(Main)
 
-[`ReportGenerator`](https://github.com/reposense/RepoSense/blob/master/src/main/java/reposense/report/ReportGenerator.java),
+[`ReportGenerator`](https://github.com/reposense/RepoSense/blob/master/src/main/java/reposense/report/ReportGenerator.java)
  1. uses `GitClone` API to clone the repository from *GitHub*.
  1. copies the template files into the designated output directory.
- 1. uses `CommitReporter` and `AuthorshipReporter` to produce the commit and authorship summary respectively.
+ 1. uses `CommitReporter` and `AuthorshipReporter` to produce the commit and authorship summary, respectively.
```
[the remark this answers is not in the document]
Comment (Tejas2805) [reply docs/dg/architecture.md:70]: Yup this one is necessary.

File: docs/about.md [hunk 3]
```diff
@@ -5,7 +5,7 @@
 
 <h1 class="display-3"><md>{{ title }}</md></h1>
 
-RepoSense is a project based in the [National University of Singapore, School of Computing](http://www.comp.nus.edu.sg/), and is funded by a _Teaching Enhancement Grant_ from [NUS Center for Development of Teaching and Learning](http://www.cdtl.nus.edu.sg/).
+RepoSense is a project-based in the [National University of Singapore, School of Computing](http://www.comp.nus.edu.sg/), funded by a _Teaching Enhancement Grant_ from the [NUS Center for Development of Teaching and Learning](http://www.cdtl.nus.edu.sg/).
```
[the remark this answers is not in the document]
Comment (Tejas2805) [reply docs/about.md:8]: How about - `project, based`

[the remark this answers is not in the document]
Comment (jamessspanggg) [reply docs/about.md:8]: no i dont think changes are required here, `RepoSense is a project based in ...` sounds fine no?

[the remark this answers is not in the document]
Comment (Tejas2805) [reply docs/about.md:8]: Yup will proceed with that

[the remark this answers is not in the document]
Comment (Tejas2805) [reply docs/about.md]: @jamessspanggg Can get your review on this to proceed?

**Comment by fzdy1914** [comment]
@jamessspanggg Your input is needed here.

**Review by jamessspanggg — APPROVED** [review]

**Comment by Tejas2805** [comment]
@fzdy1914 Can merge this.

**Pull request merged**